src/crypto/verification/SAS.js

```diff
@@ -165,6 +166,15 @@ const macMethods = {
     "hmac-sha256": "calculate_mac_long_kdf",
 };
 
+function calculateMAC(olmSAS, method) {
+    return function(...args) {
+        const macFunction = olmSAS[macMethods[method]];
+        const mac = macFunction.apply(olmSAS, args);
+        logger.log("SAS calculateMAC:", method, args, mac);
```

Comment:
Hmm, I think you told me the other day this doesn't take multiple args... 😜

Reply:
I tried now and it indeed works :smile: